core/src/test/scala/tech/beshu/ror/integration/CurrentUserMetadataAccessControlTests.scala

```diff
@@ -171,7 +171,13 @@ class CurrentUserMetadataAccessControlTests
       |    auth_key: "user2:pass"
       |
       |  - username: user4
-      |    groups: ["group5", "group6"]
+      |    groups:
+      |      - local_group:
```

Comment:
oh, little misunderstanding here. When we use `GroupMappings.Simple` there is no need to specify that it's a local group. But the `GroupMappings.Advanced` is used together with defining different group name, it's better "wrap" the local group attributes under the `local_group` key.

Reply:
So in this case the config should look like this:
```
- username: user4
  groups:
    - id: group5
      name: group 5
```